overlord/snapstate/can_remove.go

```diff
@@ -17,29 +17,26 @@
  *
  */
 
-package snappy
+package snapstate
```

Comment:
Feels strange to have a file can_remove.go by itself, by the way. Nothing else makes sense here then, so doesn't feel like the right granularity. This can be util.go or similar if there's nothing better.